packages/CollabCodeEdit-Tests.package/CCESessionTest.class/instance/testBroadcastAddedOrChangedMethodSendsCCEMessageWithCCEEdit.st

```diff
@@ -0,0 +1,4 @@
+tests
+testBroadcastAddedOrChangedMethodSendsCCEMessageWithCCEEdit
+	session broadcastAddedOrModifiedMethod: #test in: CCETestDummy.
+	self assert: (clientMock hasEditOfType: CCEMethodAddedOrModifiedEdit)
```

Comment:
i would suggest to clear the mock messages after every test, because then it can be ensured that every test finds a empty interface which makes the tests more reliable